apps/voting/app/src/components/VoteActions.js

```diff
@@ -0,0 +1,271 @@
+                  `}
+                >
+                  {connectedAccountVote === VOTE_YEA ? 'yes' : 'no'}
+                </span>{' '}
+                with{' '}
+                <span
+                  css={`
+                    color: ${theme.surfaceContent};
+                    font-weight: 600;
+                  `}
+                >
+                  {userBalance === -1 ? '…' : userBalance} {tokenSymbol}
+                </span>
+                .
+              </div>
+            </div>
+          </div>
+        )}
+      </React.Fragment>
+    )
+  }
+
+  if (canUserVote && hasVoted && !changeVote) {
+    return (
+      <div>
+        <Button
+          mode="strong"
+          onClick={handleChangeVote}
+          wide
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          Change my vote
+        </Button>
+        <TokenReference
+          userBalance={userBalance}
+          tokenSymbol={tokenSymbol}
+          snapshotBlock={snapshotBlock}
+          startDate={startDate}
+        />
+      </div>
+    )
+  }
+
+  if (canUserVote) {
+    return (
+      <div>
+        {connectedAccount ? (
+          <React.Fragment>
+            <Buttons onClickYes={onVoteYes} onClickNo={onVoteNo} />
+            <TokenReference
+              userBalance={userBalance}
+              tokenSymbol={tokenSymbol}
+              snapshotBlock={snapshotBlock}
+              startDate={startDate}
+            />
+          </React.Fragment>
+        ) : (
+          <div
+            css={`
+              border-radius: ${RADIUS}px;
+              background: ${theme.background};
+              padding: ${3.5 * GU}px ${10 * GU}px;
+              text-align: center;
+            `}
+          >
+            <div
+              css={`
+                ${textStyle('body1')};
+              `}
+            >
+              You must enable your account to vote on this proposal
+            </div>
+            <div
+              css={`
+                ${textStyle('body2')};
+                color: ${theme.surfaceContentSecondary};
+                margin-top: ${2 * GU}px;
+              `}
+            >
+              Connect to your Ethereum provider by clicking on the Enable
+              account button on the header. You may be temporarily redirected to
+              a new screen.
+            </div>
+          </div>
+        )}
+      </div>
+    )
+  }
+
+  return (
+    <div>
+      <Buttons disabled />
+      <Info mode="warning">
+        The currently connected account did not hold any {tokenSymbol} at the
+        time this vote began ({formatDate(startDate)}), and therefore cannot
+        participate in this vote. Make sure your accounts are holding{' '}
+        {tokenSymbol} at the time a vote begins if you'd like to vote using this
+        Voting app.{' '}
+        <ExternalLink
+          target="_blank"
+          href="https://wiki.aragon.org/documentation/aragon_network_token/"
```

Comment:
@dizzypaty This was a gross oversight on my part. We should not use this link, since this is specific to only `governance.aragonproject.eth`'s Voting apps.

Some ideas instead:

- Some text saying "Please ask the organization's administrators how to obtain tokens"
- A link to the token contract on etherscan

Reply:
Your suggestions are great, let's make that change.

Reply:
This was not worked on, I'll remove this an we can push to a next iteration the correct copy + link